docs/usage/getting-started/running.md

```diff
@@ -92,8 +95,15 @@ It is built similarly to the default "full" Renovate image described above, but
 - It is released every 1-2 months in a slower, more stable cadence than Renovate OSS, which releases on every commit
 - It's licensed using an end-user license agreement (EULA) and not the Affero General Public License (AGPL)
 
-WSOP supports GitHub (both `github.com` and GitHub Enterprise Server) as well as GitLab self-hosted.
-Documentation can be found in its public GitHub repository [`whitesource/renovate-on-prem`](https://github.com/whitesource/renovate-on-prem).
+Plus, the Enterprise Edition has:
+
+- Horizontal scaling to run multiple 'worker' containers
+- Dedicated support from Mend.io
+- Premium features, including Smart Merge Control
```

Comment:
Is there a link to the Smart Merge Control documentation? From reading just the docs it's not clear what this is, or why I might want to have the feature. 😄

Reply:
Will consider adding this later.